src/common-principles.md

```diff
@@ -513,6 +513,28 @@ Example:
 }
 ```
 
+Each level can be described with a string as in the example above,
+or with an object containing the fields `Description` and `TermURL`
```

Comment:
Do we expect BIDS users to know what `TermURL` expects? How about:

```suggestion
or with an object containing the fields `Description` and `TermURL`.
If you use the object format, the value of the `TermURL` key should
be a URL to a term from a controlled vocabulary that describes the
level. For example: https://www.ncbi.nlm.nih.gov/mesh/68008297 for 
"male". The value of the `Description` key should then correspond
to the human readable name of this term (or to the preferred name
if several options exist in the controlled vocabulary). For example: "Male".
```

Reply:
If we're going to improve the definition of TermURL, we should do it in https://bids-specification.readthedocs.io/en/stable/glossary.html#termurl-metadata so it appears in the glossary and any tables it's defined in.

 https://bids-specification.readthedocs.io/en/stable/glossary.html#termurl-metadata

Reply:
I could only transfer one sentence to the TermURL generic definition, the rest of @surchs suggestion feels too specific to this example I think.